compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1668,6 +1669,12 @@ pub struct TargetOptions {
     /// Whether the target supports stack canary checks. `true` by default,
     /// since this is most common among tier 1 and tier 2 targets.
     pub supports_stack_protector: bool,
+
+    // The name of entry function.
+    pub entry_name: StaticCow<str>,
+
+    // The ABI of entry function.
```

Comment:
Nit: I think it would be useful for these comments to mention the default values.